### PR TITLE
Set enemies of the 'Creeps' player when adding a new player in the editor

### LINKED ISSUE #10127: Map editor doesn't set Creeps' enemies
Creating a new map with a player results in yaml like such:

```
Players:
	PlayerReference@Neutral:
		Name: Neutral
		OwnsWorld: True
		NonCombatant: True
		Faction: england
	PlayerReference@Creeps:
		Name: Creeps
		NonCombatant: True
		Faction: england
	PlayerReference@Multi0:
		Name: Multi0
		Playable: True
		Faction: Random
		Enemies: Creeps
```

Missing `Enemies: Multi0` for Creeps

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/28670537-map-editor-doesn-t-set-creeps-enemies?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by obrakmann**
Fixes #10127.

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/World/EditorActorLayer.cs [hunk 1]
```diff
@@ -186,6 +186,10 @@ void SyncMultiplayerCount()
 				Players.Players.Add(pr.Name, pr);
 				worldRenderer.UpdatePalettesForPlayer(pr.Name, pr.Color, true);
 			}
+
+			var creeps = Players.Players.Keys.FirstOrDefault(p => p == "Creeps");
+			if (string.IsNullOrEmpty(creeps))
```
Comment (Phrohdoh) [inline OpenRA.Mods.Common/Traits/World/EditorActorLayer.cs]: Shouldn't this be negated?

Comment (obrakmann) [inline OpenRA.Mods.Common/Traits/World/EditorActorLayer.cs]: Yeah, it probably should, shouldn't it? That's what I get for doing last-minute changes.

**Comment by penev92** [comment]
:+1:

**Comment by Mailaender** [comment]
Works as promised. :white_check_mark:

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/17730acc6e789cb12ef1c44e7e9d9a11a9854e8b...32eacb7365790c95b24d68098695d49456026b3a)

**Pull request merged**